TWLight/resources/templates/resources/partner_users.html

```diff
@@ -0,0 +1,86 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+  {% comment %} Translators: This is the heading of a page listing editors who have applications for this partner ({{ object }}). {% endcomment %}
+  <h1>
+  {% blocktrans with partner=object.company_name %}
+  {{ object }} approved users
+  {% endblocktrans %}
+  </h1>
+
+  <div class="alert alert-info">
+    {% comment %} Translators: This message is shown on pages where coordinators can see personal information about applicants. {% endcomment %}
+    {% trans 'Coordinators: This page may contain personal information such as real names and email addresses. Please remember that this information is confidential.' %}
+  </div>
+
+  {% if approved_applications|length > 0 %}
+    {% comment %} Translators: On the page listing approved users for a partner, this is the title of the section listing applications with the 'approved' status {% endcomment %}
+    <h2>{% trans "Approved applications" %}</h2>
+    <table class="table table-sm">

```

Comment:
the text in this table should be marked for translation. Have you looked at this on a phone? Make use of a responsive grid instead of a table wherever possible.